sarif/operations/blame_op.py

```diff
@@ -94,10 +95,16 @@ def _enhance_with_blame(input_files, repo_path):
     print(f"Found blame information for {blame_info_count} of {item_count} results")
 
 
+def _make_path_git_compatible(file_path):
+    if file_path.startswith("file://"):
+        return urllib.request.url2pathname(file_path[7:])
```

Comment:
I'd suggest using `urllib.parse.urlparse` to get the path component here rather than attempting to parse it yourself. I know from my own team's bug tail that URI parsing is surprisingly complex. For example, UNC paths, Linux paths, etc may not behave the way you expect. Sometimes there's a third slash after `file://` depending on whether the URI contains an "authority". Etc.

Reply:
Thanks for the suggestion - I've rewritten to use `urlparse` before `url2pathname`.  Please check you are happy with the change and approve if so 😃